lochness/email/__init__.py

```diff
@@ -19,10 +29,146 @@ def send(recipients, sender, subject, message):
     s.sendmail(sender, recipients, msg.as_string())
     s.quit()
 
+
+def send_detail_google(Lochness,
+                       title: str, subtitle: str, first_message: str,
+                       second_message: str, code: List[str],
+                       in_mail_footer: str,
+                       test: bool = False) -> None:
+    '''Send email using gmail account which has it's "Less secure app" allowed
+
+    Set up an google account to send out emails using its SMTP server. You need
+    to allow "Less secure app" in Google aacount page in order to use this
+    function.
+
+    The configuration file should have
+
+        sender: senderemail@gmail.com
+        notify:
+          __global__:
+              - receiver1@anyemail.com
+              - receiver2@anyemail.com
```

Comment:
After I append a recipient to the configuration file, must `sync.py` be restarted so `send_detail_google()` has knowledge of them?

Reply:
Correct. Many other lochness functions also require the `sync.py` to be restarted once the configuration file is changed.

Reply:
Restarting won't be required if we provide:

https://github.com/AMP-SCZ/lochness/blob/75c6dd2c832f766f4bff63de864b5b3d67b824d2/scripts/send_email_update.py#L22-L25

But this `--recipients` option is not attached to `sync.py`, is it?

Reply:
`send_email_update.py` is a separate code from `sync.py`. It's not attached to `sync.py`

Reply:
Okay, I realize that `send_email_update.py` can be run separately anytime to send emails. Thank you.